services/src/integ-test/resources/uk/gov/moj/sdt/services/sql/IndividualRequestMdbIntTest.sql

```diff
@@ -1,3 +1,8 @@
+TRUNCATE TABLE BULK_CUSTOMERS CASCADE;
+TRUNCATE TABLE TARGET_APPLICATIONS CASCADE;
+TRUNCATE TABLE INDIVIDUAL_REQUESTS CASCADE;
+TRUNCATE TABLE BULK_CUSTOMER_APPLICATIONS CASCADE;
```

Comment:
Test data is inserted into BULK_SUBMISSIONS table, so should there be a TRUNCATE statement for that table as well?

Reply:
done